### PR TITLE
tasks and tests

### PR DESCRIPTION
**Pull request by Vladakkk**

### CODE REVIEW

File: src/main/java/org/example/JewelsAndStones.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            char stone = stones.charAt(i);
+            if (jewels.indexOf(stone) != -1){
+                count++;
+            }
+        }
+
+        return count;
+    }
+
+    public static void main(String[] args){
```
Comment (mkharkhuta) [inline src/main/java/org/example/JewelsAndStones.java:16]: please avoid using main as you already known about tests

File: src/test/java/org/example/JewelsAndStonesTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class JewelsAndStonesTest {
+    @Test
+    public void testCountJewelsInStones(){
+        Assert.assertEquals(3, JewelsAndStones.countJewelsInStones("aA", "aAAbbbb"));
+        Assert.assertEquals(0, JewelsAndStones.countJewelsInStones("z", "ZZ"));
+    }
```
Comment (mkharkhuta) [inline src/test/java/org/example/JewelsAndStonesTest.java:11]: this is a good approach for testing positives, negatives, and some edge cases. for example cauntJewels_JewelsEmpty_BadRequestIsThrown() {... }. Keep calling the Given-When-Then pattern. It would be clear what is being tested and what the expected result is

**Review by mkharkhuta — COMMENTED** [review]
Please, remove main methods and add more tests (positive, negative, edge cases)
